From foo@baz Tue Apr  9 12:12:43 2002
Date: Sun, 30 Jul 2006 01:34:46 -0700
To: Greg KH <greg@kroah.com>
From: Greg Kroah-Hartman <gregkh@suse.de>
Subject: Driver core: minimum version of udev is changed to 081

This is needed for the move from class_device to device by many of the
different subsystems.  udev 081 was released Jan 16, 2006, so it is an
older version (at least 6 months old), than what is currently available
and shipping in the newer distros right now.

Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 Documentation/Changes |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- gregkh-2.6.orig/Documentation/Changes
+++ gregkh-2.6/Documentation/Changes
@@ -45,7 +45,7 @@ o  isdn4k-utils           3.1pre1       
 o  nfs-utils              1.0.5                   # showmount --version
 o  procps                 3.2.0                   # ps --version
 o  oprofile               0.9                     # oprofiled --version
-o  udev                   071                     # udevinfo -V
+o  udev                   081                     # udevinfo -V
 
 Kernel compilation
 ==================
